python/lsst/ts/salobj/sal_info.py

```diff
@@ -251,6 +271,12 @@ def __init__(self, domain, name, index=0):
         self.authorized_users = set()
         self.non_authorized_cscs = set()
 
+        # Publishers and subscribers; create at need to avoid
```

Comment:
to avoid what?

Reply:
Oops. I fixed the comment and also expanded some related material.